colabfold_search.sh

```diff
@@ -11,6 +11,7 @@ DB3="$7"
 USE_ENV="$8"
 USE_TEMPLATES="$9"
 FILTER="${10}"
+THREADS="$11"
```

Comment:
shellcheck complained about this, even though it worked in my bash: https://github.com/koalaman/shellcheck/wiki/SC1037

```suggestion
THREADS="${11}"
```